### ISSUE TITLE
Error prompt is observed while trying to use windows location services to determine time.

### ISSUE THREAD

**Issue by Michellehdr** [t1m0thyj/WinDynamicDesktop#618]
- WDD Version: v5.6.0.0
- OS Version: Windows 11

Steps to Reproduce:

1.Deploy latest OS and Perform Windows and Store updates.  
2. Download the application from store and install it
3. Launch the application.
4. Select the language as English and click on ok.
5. Click on yes in the location access prompt window. 
6. Select use windows location services to determine time.
7. Click on Grant permission.
8. Click on ok.
9. Observe the error.

Actual Result: Error prompt is observed while trying to use windows location services to determine time.
![image](https://github.com/user-attachments/assets/cb906b6e-d228-42fc-a8d2-42edd9b15e2f)

Are you using the Microsoft Store version of WDD?: Yes

**Comment by Michellehdr** [comment]
@t1m0thyj I tried with the link you shared, but seems like there is no installer in that link. Could you please double check? Thanks！

**Comment by Michellehdr** [comment]
@t1m0thyj  I tried with the link you shared, but seems like there is no installer in that link. Could you please double check? Thanks！
https://github.com/t1m0thyj/WinDynamicDesktop/actions/runs/11790916909

**Comment by t1m0thyj** [comment]
> @t1m0thyj I tried with the link you shared, but seems like there is no installer in that link. Could you please double check? Thanks！ https://github.com/t1m0thyj/WinDynamicDesktop/actions/runs/11790916909

Yes the workflow does not build an installer. You can download the portable EXE for your platform and run it directly, or copy it into the folder where you have previously installed WinDD.

**Comment by Michellehdr** [comment]
Thanks, @t1m0thyj. We verified the issue with the exe you provided and found the issue no longer repro. Will the test build be available to the users?

**Comment by t1m0thyj** [comment]
Just published 5.6.1, so the fix is now available to users on GitHub and will be available on MS Store once approved.
